modules/terragrunt/stack_generate.go

```diff
@@ -0,0 +1,38 @@
+package terragrunt
+
+import (
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/gruntwork-io/terratest/modules/testing"
+)
+
+// TgStackGenerate calls terragrunt stack generate and returns stdout/stderr
+func TgStackGenerate(t testing.TestingT, options *Options) string {
+	out, err := TgStackGenerateE(t, options)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return out
+}
+
+// TgStackGenerateE calls terragrunt stack generate and returns stdout/stderr
+func TgStackGenerateE(t testing.TestingT, options *Options) (string, error) {
+	if options.TerragruntBinary != "terragrunt" {
```

Comment:
Not sure about this logic, for example if I have multiple terragrunt executables set:

/opt/terragrunt-v0.20.0
/opt/terragrunt-v0.68.2

and will set `options.TerragruntBinary=/opt/terragrunt-v0.68.2` for a test

It will throw an error :thinking: